clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -1385,14 +1393,26 @@ describe("Fides-js TCF", () => {
             fidesString: undefined,
           },
           experience: OVERRIDE.UNDEFINED,
-          // the below ensures we do not fetch experience client-side either
         },
         OVERRIDE.UNDEFINED,
-        OVERRIDE.UNDEFINED
+        OVERRIDE.EMPTY
```

Comment:
I guess EMPTY more accurately reflects the real-world here, where we'd still successfully call for client-side experience but it returns an empty response 👍